vocabularies/Common.xml

```diff
@@ -1454,6 +1454,132 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         <Annotation Term="Core.Description" String="Base URL for WebSocket connections" />
       </Term>
 
+      <Term Name="RecommendationsFunction" Type="Common.QualifiedName" Nullable="false" AppliesTo="EntityType">
```

Comment:
This annotation is not needed for the short-term orchestration, because the OData function is called in an app-specific code extension.

Reply:
Can we add this as well even in case application may not use short term?

Reply:
Why? We anyway plan to use a different annotation (`Core.ImplementsSignature`) long-term.